envs/environment.yaml

```diff
@@ -8,11 +8,11 @@ channels:
 - bioconda
 - gurobi
 dependencies:
-- python>=3.8
+- python>=3.8, <3.12  # 3.12 has weird issues with snakemake https://github.com/snakemake/snakemake/issues/2480
```

Comment:
Just for the sake of completeness: according to [this](https://github.com/snakemake/snakemake/issues/2648) Snakemake issue the problem was fixed by [this](https://github.com/snakemake/snakemake/pull/2649) PR. Though, it appears there are some deeper issues https://github.com/snakemake/snakemake/issues/3354 for which the description is even more relevant 😄 

Would you mind to replace a link to the relevant issues: `https://github.com/snakemake/snakemake/issues/2480` -> `https://github.com/snakemake/snakemake/issues/3354`?

Reply:
The one you mentioned is related to python 3.13 I believe, not 3.12. There are multiple problems related to the versioning, the one I mentioned is the one I actually encountered in the past when running the test. I'd be prone to keep that as I actually encountered that problem but not the one related to 3.13 (yet)

Reply:
Thanks for clarifying!